### PR TITLE
live: better overview in docs

### PR DESCRIPTION
**Pull request by FZambia**
**What this PR does / why we need it**:

A better overview of Grafana Live on Live documentation index page.

* move features to index since otherwise those are hidden for a reader, see  #36329, added alias to redirect from old feature description page
* add concepts section to share how things work, mention channel structure and configuration

**Which issue(s) this PR fixes**:

Addresses #36329

### CODE REVIEW

File: docs/sources/live/_index.md [hunk 1]
```diff
@@ -13,3 +13,35 @@ With Grafana Live, you can push event data to a frontend as soon as an event occ
 This could be notifications about dashboard changes, new frames for rendered data, and so on. Live features can help eliminate a page reload or polling in many places, it can stream Internet of things (IOT) sensors or any other real-time data to panels.
 
 > **Note:** By `real-time`, we indicate a soft real-time. Due to network latencies, garbage collection cycles, and so on, the delay of a delivered message can be up to several hundred milliseconds or higher.
+
+## Concepts
+
+Grafana Live sends data to clients over persistent WebSocket connection. Under the hood Grafana frontend subscribes on channels to receive data which was published into that channel – in other words PUB/SUB mechanics is used. All subscriptions on a page multiplexed inside a single WebSocket connection. There are some rules regarding Live channel names – see [Live channel]({{< relref "live-channel.md" >}}).
```
Comment (oddlittlebird) [inline docs/sources/live/_index.md:19]:
```suggestion
Grafana Live sends data to clients over persistent WebSocket connection. Grafana frontend subscribes on channels to receive data which was published into that channel – in other words PUB/SUB mechanics is used. All subscriptions on a page multiplexed inside a single WebSocket connection. There are some rules regarding Live channel names – see [Live channel]({{< relref "./live-channel.md" >}}).
```

File: docs/sources/live/_index.md [hunk 2]
```diff
@@ -13,3 +13,35 @@ With Grafana Live, you can push event data to a frontend as soon as an event occ
...
+
+Handling persistent connections like WebSocket in scale may require operating system and infrastructure tuning. That's why by default Grafana Live supports 100 simultaneous connections max, see more details on how to tune this limit in [Live configuration section]({{< relref "configure-grafana-live.md" >}}).
```
Comment (oddlittlebird) [inline docs/sources/live/_index.md:21]:
```suggestion
Handling persistent connections like WebSocket in scale may require operating system and infrastructure tuning. That's why by default Grafana Live supports 100 simultaneous connections max. For more details on how to tune this limit, refer to [Live configuration section]({{< relref "configure-grafana-live.md" >}}).
```

File: docs/sources/live/_index.md [hunk 3]
```diff
@@ -13,3 +13,35 @@ With Grafana Live, you can push event data to a frontend as soon as an event occ
...
+
+### Dashboard change notifications
+
+As soon as there is a change to the dashboard layout, it is automatically reflected on other devices connected to Grafana Live.
+
+### Data streaming from plugins
+
+With Grafana Live, backend data source plugins can stream updates to frontend panels.
+
+For data source plugin channels Grafana uses `ds` scope. Namespace in the case of data source channels is a data source unique ID (UID) which is issued by Grafana at the moment of data source creation. The path is a custom string that plugin authors free to choose themselves (just make sure it consists of allowed symbols).
```
Comment (oddlittlebird) [inline docs/sources/live/_index.md:35]:
```suggestion
For data source plugin channels, Grafana uses `ds` scope. Namespace in the case of data source channels is a data source unique ID (UID) which is issued by Grafana at the moment of data source creation. The path is a custom string that plugin authors free to choose themselves (just make sure it consists of allowed symbols).
```

**Review by oddlittlebird — APPROVED** [review]
Made a couple suggestions. Otherwise, g2g.

**Review by ryantxu — APPROVED** [review]

**Pull request merged**